docs/docs-content/troubleshooting/generate-har-files.md

```diff
@@ -0,0 +1,155 @@
+---
+sidebar_label: "Generate a HAR File"
+title: "Generate HTTP Archive Files for Troubleshooting"
+description: "Learn how to generate and sanitize HAR files to speed up issue resolution."
+sidebar_position: 70
+tags: ["troubleshooting", "har", "support"]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleStop, faTrashCan } from '@fortawesome/free-regular-svg-icons';
+import { faBan, faPause, faPlay, faGear } from '@fortawesome/free-solid-svg-icons';
+import RecordNetworkLog from '/src/components/CustomIcons/RecordNetworkLog';
+
+When you encounter page performance or rendering issues in Palette, we may ask you to generate an HTTP Archive (HAR) file to help us identify and resolve those issues.
+
+:::caution
+HAR files contain all network activity for a specific page, including sensitive and confidential information, such as API keys, secrets, cookies, passwords, and more. Before sending your HAR files to us, sanitize the sensitive data. We do not recommend using any third-party tools for HAR sanitization, as they can overlook some details.
+:::
+
+This guide explains how you can generate HAR files in Chrome, Safari, and Firefox and sanitize the sensitive data they contain.
+
+## Generate HAR Files
+
+<Tabs>
+  <TabItem value="chrome" label="Chrome" default>
+    1. Open the page with issues in Chrome.
+    2. On your keyboard, open Developer Tools:
+       - For Windows and Linux, press **F12** or **Ctrl + Shift + I**.
+       - For Mac, press **Fn + F12** or **Cmd + Option + I**.
+
+       For alternative methods to open Developer Tools, refer to [Open Chrome DevTools](https://developer.chrome.com/docs/devtools/open).
+    3. In Developer Tools, select the **Network** tab. Then, on the **Network** pane:
+       - Toggle <RecordNetworkLog /> to <FontAwesomeIcon icon={faCircleStop} color={"#E46962"} /> to start recording network activity.
+       - Click <FontAwesomeIcon icon={faBan} rotation={270} /> to clear the current network log.
+       - Select the **Preserve log** and **Disable cache** checkboxes.
+
+       ![](/assets/docs/images/troubleshooting/generate-har-files/chrome-dev-tools-settings.png)
```

Comment:
We have a image naming convention here: https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1780547845/Image+File+Naming+Standard